src/main/java/com/twilio/oai/TwilioNodeGenerator.java

```diff
@@ -110,13 +90,13 @@ public void processOpenAPI(final OpenAPI openAPI) {
     public String toApiFilename(final String name) {
         // Replace the path separator placeholder with the actual separator and lowercase the first character of each
         // path part.
-        String[] split = super.toApiFilename(name).split(PATH_SEPARATOR_PLACEHOLDER);
-        if(isPreviewDomain()){
-            return getSubDomainName(subDomainMap, name) + "/" +Arrays.stream(Arrays.copyOfRange(split, 0, split.length - 1)).map(String::toLowerCase).collect(Collectors.joining("/")) + "/"+split[split.length-1];
+        final String[] splitName = super.toApiFilename(name).split(PATH_SEPARATOR_PLACEHOLDER);
+        if (isPreviewDomain()) {
+            return getSubDomainName(subDomainMap, name) + "/" +Arrays.stream(Arrays.copyOfRange(splitName, 0, splitName.length - 1)).map(String::toLowerCase).collect(Collectors.joining("/")) + "/" + splitName[splitName.length - 1];
```

Comment:
nit: This code is hard to read (the comment above helps a bit). Can we split it to multiple lines and maybe add an example comment of the transformation?

Reply:
Agreed, but I didn't change any of the logic here; just naming. I'm working on more refactorings to reduce duplications and this is one.